pkg/oauth/apis/oauth/types.go

```diff
@@ -4,6 +4,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// +genclient
+// +genclient:nonNamespaced
```

Comment:
@ironcladlou in some resources methods were not implemented in rest (leading to 404 or worse)... maybe we should generate all verbs in client but it might be confusing to users (they use Patch() but always get 404 as it is not implemented...